src/pages/pool/_id.vue

```diff
@@ -210,106 +127,120 @@ interface PoolPageData {
 export default defineComponent({
   components: {
     ...PoolPageComponents,
-    GauntletIcon,
-    APRTooltip,
     StakingIncentivesCard,
     StakingProvider,
     ApyVisionPoolLink,
-    BalChipNew
+    PoolPageHeader
   },
 
   setup() {
     /**
      * COMPOSABLES
      */
-    const { appLoading } = useApp();
     const { t } = useI18n();
     const route = useRoute();
-    const { fNum2 } = useNumbers();
-    const { explorerLinks, isWalletReady } = useWeb3();
+    const { explorerLinks } = useWeb3();
     const { prices } = useTokens();
-    const { blockNumber, isKovan, isMainnet, isPolygon } = useWeb3();
     const { addAlert, removeAlert } = useAlerts();
-    const { balancerTokenListTokens } = useTokens();
     const { isAffected, warnings } = usePoolWarning(route.params.id as string);
 
-    /**
-     * QUERIES
-     */
-    const poolQuery = usePoolQuery(route.params.id as string);
-    const poolSnapshotsQuery = usePoolSnapshotsQuery(route.params.id as string);
-
     /**
      * STATE
      */
     const data = reactive<PoolPageData>({
       id: route.params.id as string
     });
 
-    /**
-     * COMPUTED
-     */
+    //#region pool query
```

Comment:
What are these comments for?

Reply:
This region comments helps to separate code to logical blocks. It improves readability  
+ it helps to collapse and hide sections of code in ide